Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertRebar.cs

```diff
@@ -61,27 +61,31 @@ public ApplicationObject RebarToNative(RevitRebarGroup speckleRebar)
       speckleRebar.shape.rebarType == RebarType.Standard ? RebarStyle.Standard : RebarStyle.StirrupTie;
 
     // get start and end hooks and orientations
-    var speckleStartHook = speckleRebar.startHook as RevitRebarHook;
     RebarHookType startHook = null;
     RebarHookOrientation startHookOrientation = RebarHookOrientation.Right;
-    if (speckleStartHook != null)
+    if (speckleRebar.startHook is RevitRebarHook speckleStartHook)
```

Comment:
I thought I'd seen this stuff in a PR from Connor but maybe I dreamt it :D

Reply:
PR from connnor ended in this Branch ;) so I bet it wasn't a dream 🙌🏼